http3/http_stream.go

```diff
@@ -268,3 +278,23 @@ func (s *stream) ReceiveDatagram(ctx context.Context) ([]byte, error) {
 	// TODO: reject if datagrams are not negotiated (yet)
 	return s.datagrams.Receive(ctx)
 }
+
+type warpReader struct {
+	io.Reader
+	first bool
+	cb    func()
```

Comment:
I'd prefer to have the trace here:
```suggestion
	trace *httptrace.ClientTrace
```

Reply:
fixed